Server/sprawl_calculator/calculator/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+#Url for endpoint1 and endpoint2
+urlpatterns = [
+    path('endpoint1/',endpoint1, name='endpoint1'),
```

Comment:
Thanks for this, the [first endpoint](https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L335) is not enpoint1 but `/v1/wup_index`

Reply:
path name changed as per documentation.